stemcell_builder/stages/bosh_users/apply.sh

```
@@ -36,3 +36,10 @@ cp $assets_dir/ps1.sh $chroot/etc/profile.d/00-bosh-ps1
 echo "source /etc/profile.d/00-bosh-ps1" >> $chroot/root/.bashrc
 echo "source /etc/profile.d/00-bosh-ps1" >> $chroot/home/vcap/.bashrc
 echo "source /etc/profile.d/00-bosh-ps1" >> $chroot/etc/skel/.bashrc
+
+# configure idle session shell timeout
+cat << 'EOF' > $chroot/etc/profile.d/01-tmout.sh
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)